disasterinfosite/migrations/0004_model-translation.py

```diff
@@ -132,4 +132,44 @@ class Migration(migrations.Migration):
             name='text_es',
             field=models.TextField(help_text='More information about building your supply kit. Any web address in here gets turned into a link automatically.', null=True),
         ),
+        migrations.AddField(
+            model_name='snuggetsection',
+            name='display_name',
+            field=models.CharField(blank=True, help_text='The name to show for this section', max_length=50),
+        ),
+        migrations.AddField(
+            model_name='snuggetsection',
+            name='display_name_en_us',
```

Comment:
Will this need to be switched to just 'display_name_en' in light of #61 ?
(sorry I didn't see these in order which would have saved this hassle)